internal/analytics/constants/constants.go

```diff
@@ -148,39 +148,47 @@ const ActCommandInputError = "command-input-error"
 // ActExecutorExit is the event action used for executor exit codes
 const ActExecutorExit = "executor-exit"
 
-// UpdateLabelSuccess is the sent if an auto-update was successful
+// UpdateLabelSuccess is sent if an auto-update was successful
 const UpdateLabelSuccess = "success"
 
-// UpdateLabelFailed is the sent if an auto-update failed
+// UpdateLabelFailed is sent if an auto-update failed
 const UpdateLabelFailed = "failure"
 
-// UpdateLabelTrue is the sent if we should auto-update
+// UpdateLabelTrue is sent if we should auto-update
 const UpdateLabelTrue = "true"
 
-// UpdateLabelForward is the sent if we should not auto-update as we are forwarding a command
+// UpdateLabelForward is sent if we should not auto-update as we are forwarding a command
 const UpdateLabelForward = "forward"
 
-// UpdateLabelUnitTest is the sent if we should not auto-update as we are running unit tests
+// UpdateLabelUnitTest is sent if we should not auto-update as we are running unit tests
 const UpdateLabelUnitTest = "unittest"
 
-// UpdateLabelConflict is the sent if we should not auto-update as the current command might conflict
+// UpdateLabelConflict is sent if we should not auto-update as the current command might conflict
 const UpdateLabelConflict = "conflict"
 
-// UpdateLabelDisabledEnv is the sent if we should not auto-update as the user has disabled auto-updates via the environment
+// UpdateLabelDisabledEnv is sent if we should not auto-update as the user has disabled auto-updates via the environment
 const UpdateLabelDisabledEnv = "disabled-env"
 
-// UpdateLabelDisabledConfig is the sent if we should not auto-update as the user has disabled auto-updates via the config
+// UpdateLabelDisabledConfig is sent if we should not auto-update as the user has disabled auto-updates via the config
 const UpdateLabelDisabledConfig = "disabled-config"
 
-// AutoUpdateLabelDisabledCI is the sent if we should not auto-update as we are on CI
+// AutoUpdateLabelDisabledCI is sent if we should not auto-update as we are on CI
 const UpdateLabelCI = "ci"
 
-// UpdateLabelFreshInstall is the sent if we should not auto-update as we are on a fresh install
+// UpdateLabelFreshInstall is sent if we should not auto-update as we are on a fresh install
 const UpdateLabelFreshInstall = "fresh-install"
 
-// UpdateLabelLocked is the sent if we should not auto-update as the state tool is locked
+// UpdateLabelLocked is sent if we should not auto-update as the state tool is locked
 const UpdateLabelLocked = "locked"
 
+// UpdateLabelSkipChecks is sent if we should not auto-update because the command explicitly skips
+// auto update checks.
+const UpdateLabelSkipChecks = "skip-checks"
```

Comment:
Do new labels require us to update the analytics reference? Can you please follow-up with Nathan after merging.

Reply:
I'm going to flag Nathan for review since this feels like some architecture changes in addition to the label thing.

Reply:
Yes we should. I've updated the doc.